### ISSUE TITLE
cell profiler features

### ISSUE THREAD

**Issue by ethancohen123** [cytomining/pycytominer#255]
Is there a simple way to extract the whole set of cell profiler features from an image using this package ?

**Comment by gwaybio** [comment]
No, this package is for processing features that have already been extracted from an image using cellprofiler (or deepprofiler)

**Comment by d33bs** [comment]
@ethancohen123 - you might be interested in https://github.com/afermg/cp_measure , a project which provides the ability to extract CellProfiler features from images. Closing this for now as this isn't something `pycytominer` is focused on.
